app/src/main/res/values-v21/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think there is a need to add this & the values-w820dp/strings.xml file. 
You can remove these two.